package.json

```diff
@@ -46,11 +48,13 @@
     "gruntfile-editor": "^0.2.0",
     "inquirer": "^0.8.0",
     "isbinaryfile": "^2.0.0",
+    "istextorbinary": "^1.0.2",
```

Comment:
As the conflicter handles buffers (he doesn't know of the actual template file), I'd drop `isbinaryfile`and only use `istextorbinary`.